app/utils/getEntityType.ts

```diff
@@ -8,7 +8,7 @@ const entityTypeMappings = {
   'meetings.meeting': 'meetings',
   'gallery.gallerypicture': 'galleryPictures',
 };
-type ServerName = $Keys<typeof entityTypeMappings>;
-export default function getEntityType(serverName: ServerName): string {
+export type EntityServerName = $Keys<typeof entityTypeMappings>;
```

Comment:
It's imported and used in `app/store/utils/contentTarget.ts` for the type defintion of `ContentTarget`. Not from this PR though, there was just an error because it wasn't exported.